src/pspm_cfg/pspm_cfg_pp_pupil.m

```diff
@@ -13,6 +13,6 @@
 cfg        = cfg_repeat;
 cfg.name   = 'Pupil & Eye tracking';
 cfg.tag    = 'pp_pupil';
-cfg.values = {pspm_cfg_process_illuminance, pspm_cfg_find_valid_fixations, pspm_cfg_pupil_correct, pspm_cfg_pupil_preprocess, pspm_cfg_pupil_size_convert, pspm_cfg_gaze_convert};
+cfg.values = {pspm_cfg_process_illuminance, pspm_cfg_find_valid_fixations, pspm_cfg_pupil_correct, pspm_cfg_pupil_preprocess, pspm_cfg_pupil_size_convert, pspm_cfg_gaze_convert, pspm_cfg_gaze_pp};
```

Comment:
In the GUI, there are diverse naming style, including "Preprocessing SCR", "Preprocess respiratory data", "Preprocess heart data", "Pupil processing" and "Preprocess gaze". Perhaps it is useful to make them consistent. I think "Preprocess XX" or "XX preprocessing" may be good choices.

Reply:
good idea - just pushed.